src/schema/src/transaction_pool.rs

```diff
@@ -243,3 +243,90 @@ where
 lazy_static! {
     pub static ref POOL: Pool = Pool::new();
 }
+
+
+#[cfg(test)]
+mod tests_transaction_pool {
+
+    use super::*;
+    use std::collections::HashMap;
+    use utils::crypto::keypair::{CryptoKeypair, Keypair, KeypairType};
+    use utils::serializer::{deserialize, serialize, Deserialize, Serialize};
+    use std::time::SystemTime;
+    pub use sdk::signed_transaction::SignedTransaction;
+
+    #[derive(Debug, Clone, PartialEq, Serialize, Deserialize, BinaryValue, ObjectHash)]
+    #[binary_value(codec = "bincode")]
+    pub struct CryptoTransaction {
+        pub from: std::string::String,
+        pub fxn_call: std::string::String,
+        pub payload: std::vec::Vec<DataTypes>,
+    }
```

Comment:
Can we avoid structure in transaction pool, since it is not dependent upon app transaction structure?